src/commands/delete-all-terms.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Delete All Terms of a given taxonomy
+ *
+ * @param taxonomy - Taxonomy to empty
+ *
+ * @example
+ * Delete all categories (note that Uncategorized term is protected)
+ * ```
+ * cy.deleteAllTerms()
+ * ```
+ *
+ * @example
+ * Delete all tags
+ * ```
+ * cy.deleteAllTerms('post_tag')
+ * ```
+ */
+export const deleteAllTerms = (taxonomy = 'category'): void => {
+  cy.visit(`/wp-admin/edit-tags.php?taxonomy=${taxonomy}`);
+  cy.get('#cb-select-all-1').click();
+
+  cy.get('body').then($body => {
+    if ($body.find('#bulk-action-selector-top').length !== 0) {
+      cy.get('#bulk-action-selector-top').select('delete');
+      cy.get('#doaction').click();
+    }
+  });
```

Comment:
What happens if we have pagination? I guess it won't delete the terms on the next pages. Do you think we should cover that case in this PR?

Reply:
Thank you for noticing that, will add iterate deletion of next pages